framework/source/class/qx/bom/webfonts/Manager.js

```diff
@@ -384,17 +392,11 @@ qx.Class.define("qx.bom.webfonts.Manager", {
     {
       var formats = qx.bom.webfonts.Manager.FONT_FORMATS;
       var sourcesMap = {};
+      var reg = new RegExp("\.(" + formats.join("|") + ")");
```

Comment:
A bit more 'fancy' RegExp could be used, which extracts the SVG-font-ID (**No** real **benefit**, as that isn't used here anyhow)... just for the record ...
```javascript
var reg = new RegExp("\.(" + formats.join("|") + ")(?:$|(?:#(.+)))");
````


```javascript
"custom/fonts/yanonekaffeesatz-regular-webfont.woff"
"custom/fonts/yanonekaffeesatz-regular-webfont.woff2"
"custom/fonts/yanonekaffeesatz-regular-webfont.svg#YanoneKaffeesatzRegular"
"custom/fonts/yanonekaffeesatz-regular-webfont.eot"
"custom/fonts/yanonekaffeesatz-regular-webfont.ttf"
```
=>
```javascript
[ ".woff"                       , "woff" , undefined                 ]
[ ".woff2"                      , "woff2", undefined                 ]
[ ".svg#YanoneKaffeesatzRegular", "svg"  , "YanoneKaffeesatzRegular" ]
​​[ ".eot"                        , "eot"  , undefined                 ]
​[ ".ttf"                        , "ttf"  , undefined                 ]
```